CHANGELOG.md

```diff
@@ -210,6 +210,8 @@ v1.11.0-rc.3
 
 - Fix panic in `loki.write` when component is shutting down and `external_labels` are configured. (@kalleep)
 
+- (_Experimental_) Remove `backend_time` field from query samples, and add `user` field to wait events within `database_observability.postgres` `query_samples` collector.
```

Comment:
Can we move this under a top level header like we have done previously
`- (Experimental) Additions to experimental database_observability.postgres component:`